src/Menu/Menu.component.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from 'react';
+import _ from 'lodash';
+import { Themes } from '../themes';
+import styled, { ThemeProvider } from 'styled-components';
+import { Popover } from '../Popover';
+
+export interface MenuItem {
+  disabled?: boolean;
+  id?: any;
+  label: string;
+}
+
+export interface Props {
+  /**
+   * An array of `MenuItem`.
+   * Each `MenuItem` can contain the following properties:
+   * - `disabled` (boolean),  *optional*
+   * - `id` (any),  *optional*
+   * - `label` (string), **required**
+   *
+   * @default []
+   **/
+  items?: MenuItem[];
+  /**
+   * Callback that's triggered when an item is clicked. Returns the original `MenuItem` and the `event` object.
+   *
+   * @default void
+   * */
+  onItemClick?(item: MenuItem, e: React.MouseEvent<HTMLElement>): void;
+  /**
+   * From theme provider
+   *
+   * @default defaultTheme
+   **/
+  theme?: any;
+  /**
+   * Optional component, such as a `Link` or `Button`, that triggers the menu's display.
+   *
+   * @default []
```

Comment:
This default type value needs to be updated since this is rendering a JSX component